tests/console/verify_and_prepare_install.pm

```diff
@@ -0,0 +1,26 @@
+# SUSE's openQA tests
+#
+# Copyright © 2017 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: check if iso was cloned and switch to X
+# Maintainer: Joachim Rauch <jrauch@suse.com>
+# Tags: boo#1040749
+
+
+use base 'opensusebasetest';
+use strict;
+use testapi;
+
+sub run {
+    assert_script_run 'ls /dev | grep vd*';
+    assert_script_run 'ls /mnt/';
+    type_string "logout\n";
+    select_console 'x11';
```

Comment:
Then define `post_run_hook` method in `copy_iso_to_external_drive.pm` and the one from `consoletest`, won't be imported. The content of that hook would be:
```
assert_script_run 'ls /dev | grep vd*';
assert_script_run 'ls /mnt/';
type_string "logout\n";
select_console 'x11'; 
```
Or keep the hook empty and have the four lines in `run`.